_playwright-tests/UI/RepositoryPackages.spec.ts

```diff
@@ -0,0 +1,95 @@
+import { test, expect } from '@playwright/test';
+import { navigateToRepositories } from './helpers/navHelpers';
+import { deleteAllRepos } from './helpers/deleteRepositories';
+import { closePopupsIfExist, getRowByNameOrUrl } from './helpers/helpers';
+export const randomName = () => (Math.random() + 1).toString(36).substring(2, 6);
+export const repoName = `${randomName()}`;
+export const edited_repo = `${repoName}-Edited`;
+export const rank = () => Math.floor(Math.random() * 10 + 1).toString();
+export const randomNum = () =>
+  Math.floor(Math.random() * 10 + 1)
+    .toString()
+    .padStart(2, '0');
+
+test.describe('Snapshot Package Count and List', async () => {
+  test('Verify package count and search functionality in snapshot details', async ({ page }) => {
+    await navigateToRepositories(page);
+    await closePopupsIfExist(page);
+    await deleteAllRepos(page);
```

Comment:
we defo don't want to delete all repos here, just delete the ones related to your test ✖️
you can delete specific repos by using the second param of the `deleteAllRepos()` function, check out other tests that use this 😄